src/common/exceptions/pagination/pagination.input.exception.ts

```diff
@@ -0,0 +1,8 @@
+import { LogContext, AlkemioErrorStatus } from '@common/enums';
+import { BaseException } from '../base.exception';
+
+export class PaginationInputException extends BaseException {
```

Comment:
The Error status and exception name have different granularity - use either of the two.